tests/data/disposedBeforeAsyncRun/MultipleUsesBeforeAsync.fs

```diff
@@ -0,0 +1,12 @@
+module M
+
+open System
+
+let asyncReturningFunc () =
+    // should warn
+    use t1 = new IO.FileStream("x", IO.FileMode.Open)
+    // should warn
+    use t2 = new IO.FileStream("x", IO.FileMode.Open)
+    async {
```

Comment:
@Smaug123 do we need to detect any usage of `t1` or `t2` inside the computation expression here?

Reply:
I guess for a first pass I'd say no. It's *definitely* a bug if they appear inside the computation expression, rather than merely "probably a bug"; but I think it's fine to be a bit over-zealous here and make the user be explicit about scopes if they intend it. For example, if the user really did intend the above code, they should instead write:

```fsharp
let asyncReturningFunc () =
    do
        use t1 = new blah
        use t2 = new blah
        ()
    async {
        ...
    }
```